docs/proposals/opaque-integer-resources.md

```diff
@@ -0,0 +1,377 @@
+# Table of Contents
+  * [Accounting and Consuming Opaque Node-Level Integer Resources](#accounting-and-consuming-opaque-node-level-integer-resources)
+    * [Terms](#terms)
+      * [Opaque node-level resource](#opaque-node-level-resource)
+    * [Requirements](#requirements)
+    * [Concrete use cases](#concrete-use-cases)
+    * [Goals](#goals)
+    * [Non-goals](#non-goals)
+    * [Design](#design)
+      * [Denoting opaque integer resources](#denoting-opaque-integer-resources)
+      * [Associating opaque integer resources with nodes](#associating-opaque-integer-resources-with-nodes)
+      * [Updates to capacity/allocatable that invalidate bound pods.](#updates-to-capacityallocatable-that-invalidate-bound-pods)
+        * [Scenario](#scenario)
+        * [Options](#options)
+      * [Ownership of NodeStatus updates](#ownership-of-nodestatus-updates)
+    * [PoC implementation - detailed changes](#poc-implementation---detailed-changes)
+      * [Changes to Kubelet](#changes-to-kubelet)
+      * [Requesting opaque integer resources for a pod](#requesting-opaque-integer-resources-for-a-pod)
+      * [Implementation](#implementation)
+        * [Notable consequences](#notable-consequences)
+    * [Related issues](#related-issues)
+    * [Usage Summary](#usage-summary)
+      * [Advertising Opaque Integer Resources](#advertising-opaque-integer-resources)
+      * [Consuming Opaque Integer Resources](#consuming-opaque-integer-resources)
+    * [Example](#usage-example)
+
+# Accounting and Consuming Opaque Node-Level Integer Resources
+
+## Terms
+### Opaque node-level resource
+A resource associated with a node that has a consumable capacity, which is handled uniformly by the scheduler and Kubelet. (If there are specializations that hang off of the resource name, then it is not opaque!)
+
+## Requirements
+1. Associate available opaque integer resources with existing node(s).
+2. Request opaque integer resources for a pod.
+3. The scheduler must account opaque node-level integer resources similarly to existing first-class resources like logical CPU cores and main memory. Co-scheduled pods must not exceed the allocatable amount for any opaque integer resource.
+
+## Concrete use cases
+Even before implementing isolation, there is already user value in accounting opaque integer resources in the node.
+
+* Exclusive access to discrete co-processors (GPGPUs, MICs, FPGAs, etc.)
```

Comment:
You would need to nix the isolation bit above for this to be true.   In general, I don't think you need the clause above.